README.md

```diff
@@ -110,7 +90,8 @@ Run this command:
 ### Usage
```

Comment:
- [ ] Your setup as shown below points the user to your microverse portfolio project rather it should point the user to your todolist project. I kindly suggest you make this change to your readme file 👍 
- [ ] Also your install section should be corrected as well
![setup](https://github.com/Errytagedesign/microverse-todo-app/assets/89055502/adb14d51-6a42-4ea6-8024-15c16e548831)